src/config/cmdline.c

```diff
@@ -392,6 +394,17 @@ remote_cmd(struct option *o, unsigned char ***argv, int *argc)
 		remote_session_flags = SES_REMOTE_RELOAD;
 		break;
 
+	case REMOTE_METHOD_SEARCH:
+		remote_session_flags = SES_REMOTE_SEARCH;
+		if (remote_argc < 1)
+			remote_url = strdup("search:");
+		else {
+			remote_url = malloc(strlen("search:") + strlen(remote_argv[0]) + 1);
+			strcpy(remote_url, "search:");
+			strcat(remote_url, remote_argv[0]);
```

Comment:
You do not check for return value of malloc. See REMOTE_METHOD_INFOBOX and please do similar. In elinks instead of strdup is called stracpy.